tests/data/validator/parameterized_message_excess_parameter.yaml

```diff
@@ -0,0 +1,4 @@
+Length: 2
+Has_Tag: True
+Tag_Value: Tag_A
+Excess: 42
```

Comment:
Is there a reason for not putting those samples into `tests/data/validator/parameterized/message/invalid`?

Reply:
Not really. Changed in a88d4b83d8dbac032e51ba31ef3583d025660ced.